docs/graphcache/gotchas.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Gotchas
+order: 5
+---
+
+# Gotchas
+
+When using the normalised cache there are a few out of the box features
+we need to be aware of, being aware of these features will make it easier
+to define best practices and to avoid pitfalls.
+
+Most of the measures we'll cover are brought to life to ensure we can build out
+to a fully offline normalised cache.
+
+## Commutativity
+
+By default the cache ensures idempotent and commutative results, the last part of this
+is the most relevant one to be aware of.
+
+Commutative guarantees means that the cache will keep the order of operations consistent, let's look
+at an example to clear this up.
+We are in our application and we queue up two queries, an `authorsQuery` and a `todosQuery`.
+
+We dispatch the `todosQuery` first and the `authorsQuery` later, the `authorsQuery` returns a result
+before the `todosQuery`. The cache will put this result from the `authorsQuery` in a
```

Comment:
```suggestion
We dispatch the `todosQuery` first and the `authorsQuery` later, but because of network latency the `authorsQuery`'s result arrived before the `todosQuery`'s result.
The cache will put the out-of-order result from the `authorsQuery` onto a layer.
```